CefSharp.WinForms/ChromiumWebBrowser.cs

```diff
@@ -273,25 +288,76 @@ private static void OnApplicationExit(object sender, EventArgs e)
         /// <summary>
         /// Initializes a new instance of the <see cref="ChromiumWebBrowser"/> class.
         /// </summary>
+        public ChromiumWebBrowser()
+        {
+
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ChromiumWebBrowser"/> class.
+        /// </summary>
         /// <param name="address">The address.</param>
-        /// <exception cref="System.InvalidOperationException">Cef::Initialize() failed</exception>
         public ChromiumWebBrowser(string address)
         {
-            if (!Cef.IsInitialized && !Cef.Initialize())
+            Address = address;
+        }
+
+        private void Initialize()
```

Comment:
> It initializes stuff, i'm not a creative person. Name suggestion?

It doesn't need to be creative, just descriptive. 

> Keeping the managed calls in the constructor results in the designer crashing.

The purely managed classes in the `CefSharp` assembly?

> But seems you want it to be DockStyle.Fill by default

What about everyone creating the control in code as they do now? Needs to be the same as it is now.